flow/network.h

```diff
@@ -351,6 +350,12 @@ class IEventFD : public ReferenceCounted<IEventFD> {
 	virtual Future<int64_t> read() = 0;
 };
 
+struct SendBuffer {
```

Comment:
Seems like this is WIP that got in by mistake?

Reply:
Yes; a second half is coming soon.  I had to move some stuff round to make things compile, and didn't want to revert that work (touching this file recompiles the world...)